src/app/components/Student.js

```diff
@@ -1,5 +1,56 @@
-import React from 'react';
+import React, { useState } from 'react';
+import Grid from '@material-ui/core/Grid';
+import FormControl from '@material-ui/core/FormControl';
+import TextField from '@material-ui/core/TextField';
+import DateFnsUtils from '@date-io/date-fns';
+import { DatePicker, MuiPickersUtilsProvider } from 'material-ui-pickers';
+import EventIcon from '@material-ui/icons/Event';
+import Button from '@material-ui/core/Button';
 
-const Student = () => <h1>Student</h1>;
+const Student = () => {
+  const [selectedDate, handleDateChange] = useState(new Date());
+  const [name, handleName] = useState('');
+
+  return (
+    <Grid container justify="center">
+      <Grid item xs={10} sm={8} md={6}>
+        <div style={{ padding: 50 }}>
```

Comment:
I think we can remove this padding so that all the pages will have the same width :) i know this is just a WIP so we can always change this later

Reply:
I didn't dismiss this! It has been updated in the file :)